Tests/SonarScanner.MSBuild.PreProcessor.Test/JreCaching/UnpackerFactoryTests.cs

```diff
@@ -23,7 +23,8 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using NSubstitute;
 using SonarScanner.MSBuild.Common;
-using SonarScanner.MSBuild.PreProcessor.JreCaching;
+using SonarScanner.MSBuild.PreProcessor.Interfaces;
```

Comment:
You can recreate the same folder structure we have in production code, moving the Unpacking classes to Unpacking.
The checksum tests can also be moved.